tools/README.md

```diff
@@ -0,0 +1,37 @@
+## How to Run Python Scripts
+
+### On your machine install Python (version 3.11.3 or above).
+
+- Windows: https://www.python.org/downloads/windows/
+- iOS:
+- Linux:
+
+### Set up for Visual Studio Code
+
+- Open project in VS Code:
+
+  1. Git Bash terminal window
+  2. Go to the location of the project
+  3. Type: code .
+
+- Create virtual environment
+  1. Open Command Palette (View->Command Palette)
+  2. Select Python: Create Environment
+  3. Select Venv (Creates a .venv virtual envir.)
```

Comment:
I guess you could have template for local.evenv (you can add your or other example) So people can just rename to dev, or anything else.

Reply:
I will prape a template for these steps above.